src/components/App.tsx

```diff
@@ -870,8 +730,8 @@ class App extends React.Component<ExcalidrawProps, AppState> {
         prevProps.offsetTop !== this.props.offsetTop)
     ) {
       this.setState({
-        width: this.props.width,
-        height: this.props.height,
+        width: this.props.width || window.innerWidth,
+        height: this.props.height || window.innerHeight,
```

Comment:
Let's use `??` here... I can think of someone wanting to do some animations and setting the dimensions to `0` at start.